src/i18n/en-events.ts

```diff
@@ -73,7 +79,7 @@ const eventsMessageDescriptors = {
       ${ColonyAndExtensionsEvents.MotionEscalated} {{agent} escalated motion {motionId} from {domain} to {newDomain}}
       ${ColonyAndExtensionsEvents.MotionRewardClaimed} {{agent} claimed their stake in motion {motionId}}
       ${ColonyAndExtensionsEvents.MotionEventSet} {Motion {motionId} fast-forwarded to the next lifecycle}
-      ${ColonyAndExtensionsEvents.ArbitraryReputationUpdate} {{agent} smote {recipient} with a {reputationPenalty} reputation penalty}
+      ${ColonyAndExtensionsEvents.ArbitraryReputationUpdate} {{agent} smote {recipient} with a {reputationChange} reputation penalty}
```

Comment:
I think it should be here smote or awarded as well depending on the case. Below is actually an award action that should up as smite:
<img width="798" alt="Screenshot 2021-11-10 at 09 42 50" src="https://user-images.githubusercontent.com/34057551/141079541-e9c0d2db-e1d5-4900-b636-f33159a3aaa8.png">

Also it seems to me that the events lit copy is missing.

<img width="1207" alt="Screenshot 2021-11-10 at 09 44 55" src="https://user-images.githubusercontent.com/34057551/141079842-209babb1-cadb-4d65-8561-d8aaf2f9c14c.png">

Reply:
About the first thing you mention, I think the responsibility of changing that might fall on to #2702 as that is the issue that should wire up the award action/motion to the action/event list, right?

Reply:
fair enough :)